### ISSUE TITLE
Issue with the blending method type: Disk-blended focal surfaces

### ISSUE THREAD

**Issue by blvnkexp** [CAOR-MINES-ParisTech/colibri-vr-unity-package#7]
Hi, I'm having an issue with the blending method type: Disk-blended focal surfaces. It simply does not do anything. I've gone through your tutorial several times thinking maybe I missed something. But still nothing. Not entirely sure why It's not working, is there any solution to this problem. Thanks

**Comment by blvnkexp** [comment]
Hi, I'm having an issue with the blending method type: Disk-blended focal surfaces. It simply does not do anything. I've gone through your tutorial several times thinking maybe I missed something. But still nothing. Not entirely sure why It's not working, is there any solution to this problem. Thanks

**Comment by DinechinGreg** [comment]
Ah, that is an issue. What platform / OS are you on? Is there a command buffer on the camera component  of the main camera ? If you open Frame Debug (Window->Analysis->Frame debugger) and click Enable, does DiskBlendedFocalSurfaces appear somewhere under Camera.Render?

**Comment by blvnkexp** [comment]
Wow, thanks for the really quick response. Thought I was going to wait a day or two minimum. I'm using windows 7. I checked the Frame Debug and there's no DiskBlendFocalSurfaces.

**Comment by blvnkexp** [comment]
I made a mistake. It's the Disk-Blended per-view meshes that's not working. Sorry about that.

**Comment by DinechinGreg** [comment]
No problem!
In the same way, on the Camera component, are any command buffers listed? (at the bottom of the component's UI)
In this case, a DiskBlendedPerViewMeshes command buffer should appear upon launching play mode.

**Comment by blvnkexp** [comment]
Yes, it does appear upon launch, but in the scene nothing is showing. I'll send a screen shot as well.
![01](https://user-images.githubusercontent.com/68740838/88398992-7ba74580-cdc6-11ea-86d5-1170f8be42e5.jpg)

**Comment by DinechinGreg** [comment]
Thanks for the screenshot!
What happens if you try "Textured per-view meshes"?

**Comment by blvnkexp** [comment]
Nothing happening.

**Comment by DinechinGreg** [comment]
In that case the meshes themselves are probably the issue. Do the depth maps seem correct? Using which checkbox did you generate the per-view meshes? Were there any errors during processing?

**Comment by blvnkexp** [comment]
The depth pass seemed to come out just as expected. I used Texture array from color data, Per-view meshes from underlying focal surfaces and per-view meshes from depth maps by quadtree simplificationand triangle removal. There won't any errors when processing.

**Comment by blvnkexp** [comment]
Is there anything specific that needs to be done to the mesh?

**Comment by blvnkexp** [comment]
I'm using unity 2019.4.0f1. Normal unity scene, so no HDRP or URP. Don't know if any of this might help narrow down a possible problem.

**Comment by blvnkexp** [comment]
Update on the issue. This is an error I get now.
![bandicam 2020-07-25 15-54-51-581](https://user-images.githubusercontent.com/68740838/88458597-76fa9400-ce8f-11ea-9c28-d06122505144.jpg)

**Comment by DinechinGreg** [comment]
Hmm, this is strange. Nothing specific should be done to the mesh, and it should work fine with your Unity configuration, so I'm not sure how to narrow down the issue.

Thanks for sharing the parsing error! Your platform probably expects "0.3" instead of "0,3", so if you change those accordingly the error should disappear. I think this issue is linked to the way we write the additional setup information, I'll patch this ASAP. However I don't think that this will solve the issues you're having afterwards during processing.

Were you working from a virtual 3D acquisition or from a set of real input data?

I'll try investigating further to see if I can reproduce the problem. Sorry about not having a clear answer yet.

**Comment by blvnkexp** [comment]
I'm using 3d renders.
